pkg/helm/helm.go

```diff
@@ -246,15 +248,27 @@ data:
 	return pkgReadWriter.Write(newNodes)
 }
 
+func (h *Hydrator) helm(ctx context.Context, args ...string) ([]byte, error) {
+	var allArgs []string
+	allArgs = append(allArgs, args...)
+	if h.CACertFilePath != "" {
+		allArgs = append(allArgs, "--ca-file", h.CACertFilePath)
+	}
+	out, err := exec.CommandContext(ctx, "helm", allArgs...).CombinedOutput()
```

Comment:
do we want the logging from shell.ExecWithDebug here?

Reply:
This isn't part of the e2e test scaffolding (where `shell.ExecWithDebug` is from), this is the helm-sync code

I don't think it's always safe to log the args here due to credentials being passed via command line args.